pkgs/by-name/fg/fgqcanvas/package.nix

```diff
@@ -0,0 +1,52 @@
+{ fetchgit
+, pkg-config
+, stdenv
+, lib
+, pkgs
+# Package dependencies
+, libsForQt5
+}:
+
+stdenv.mkDerivation rec {
+  pname = "fgqcanvas";
+  version = "0.1";
```

Comment:
how d'ya know it's that version? since it's in the flightgear repo, the git tags are from flightgear. if it doesn't actually have semantic versioning then it should be `unstable-<date-of-commit>`